setup.py

```diff
@@ -93,7 +93,17 @@
         numpy_include = np.get_numpy_include()
 
     # Determine if we have an available BLAS implementation
-    if 'bdist_wheel' not in sys.argv:
+    if platform.system() == 'Windows':
+        try:
+            with open(os.devnull, 'w') as devnull:
+                exitCode = subprocess.check_output(
+                    "python tools/python/blas/check_cblas.py build_ext",
+                    stderr=devnull,
+                    shell=True)
```

Comment:
Add a comment: 
If blas is found we will be able to link tick against it

Reply:
good question - I've never installed BLAS on windows so I have no idea.

If they threw the blas.lib file into the python lib directory then sure that could work